packages/page-settings/src/I18n/index.tsx

```diff
@@ -65,12 +65,12 @@ async function retrieveAll (): Promise<Defaults> {
   const missing = keys.filter((lng) => !languageCache[lng]);
   const english = await retrieveEnglish();
   const translations = missing.length
-    ? await Promise.all(missing.map((lng) => retrieveJson(`${lng}/translation.json`)))
+    ? await Promise.allSettled(missing.map((lng) => retrieveJson(`${lng}/translation.json`)))
     : [];
 
   // setup the language cache
   missing.forEach((lng, index): void => {
-    languageCache[lng] = translations[index] as Record<string, string>;
+    languageCache[lng] = translations[index] as unknown as Record<string, string>;
```

Comment:
This is not quite correct. So `allSettled` returns either one of these two objects -

- `{status: "fulfilled", value: <translation value> }`
- `{status: "rejected",  reason: Error: <an error> }`

So I'm guessing it probably should be something like -

```js
const { value } = translations[index];

if (value) {
  languageCache[value] = value as Record<string, string>;
}

Reply:
ok thanks i'll check that

Reply:
i edited the PR and made it simpler
no changes in typing anymore